packages/styles/src/variables/components/_tag.scss

```
@@ -0,0 +1,20 @@
+@use './../color' as col;
+@use './../commons' as commons;
+@use './../spacing' as spacing;
+@use './../type' as type;
+
+$tag-max-width: 296px;
+$tag-border-radius: spacing.$size-micro;
+$tag-border-width: commons.$border-width;
+$tag-gap: spacing.$size-micro;
+$tag-font-size: type.$font-size-16;
+$tag-horizontal-padding: spacing.$size-mini;
```

We usually use "padding-x" and "padding-y":
```suggestion
$tag-padding-x: spacing.$size-mini;
```